WMF Framework/SharedContainerCache.swift

```diff
@@ -1,35 +1,42 @@
 import Foundation
 
-public final class SharedContainerCache<T: Codable> {
-    
-    // values must all be distinct
-    public enum PathComponent: String {
-        case widgetCache = "Widget Cache"
-        case pushNotificationsCache = "Push Notifications Cache"
-        case userDataExportSyncInfo = "User Data Export Sync Info"
-    }
-    
-    private let pathComponent: PathComponent
+public struct SharedContainerCacheCommonFileNames {
```

Comment:
Actually since `talkPageCache` technically becomes a folder, maybe just name this `SharedContainerCacheCommonNames` instead of `SharedContainerCacheCommonFileNames`.